Client/src/Pages/Register/index.jsx

```diff
@@ -91,16 +92,26 @@ const Register = () => {
       }
     } catch (error) {
       if (error.name === "ValidationError") {
-        // TODO Handle validation errors
-        console.log(error);
-        alert(error);
+        // validation errors
+        createToast({
+          variant: "info",
+          body: error.details[0].message,
```

Comment:
We should do a check here before trying to access items from the `details` array, it's not guaranteed to be populated